test/data-dictionary_test.js

```diff
@@ -22,39 +22,55 @@ describe('findField', () => {
 describe('isDateField', () => {
   it('returns true for fields with date validation', () => {
     const validationProperty = 'text_validation_type_or_show_slider_number';
-    expect(isDateField({[validationProperty]: ""})).toBe(false);
-    expect(isDateField({[validationProperty]: "date_mdy"})).toBe(true);
-    expect(isDateField({[validationProperty]: "date_mdy"})).toBe(true);
-    expect(isDateField({[validationProperty]: "date_something"})).toBe(true);
-    expect(isDateField({[validationProperty]: "datetime_dmy"})).toBe(true);
+    expect(isDateField({ [validationProperty]: '' })).toBe(false);
+    expect(isDateField({ [validationProperty]: 'date_mdy' })).toBe(true);
+    expect(isDateField({ [validationProperty]: 'date_mdy' })).toBe(true);
+    expect(isDateField({ [validationProperty]: 'date_something' })).toBe(true);
+    expect(isDateField({ [validationProperty]: 'datetime_dmy' })).toBe(true);
   });
 });
 
 describe('getDateFields', () => {
   it('returns fields with date validation', () => {
-    const dateFields = [exampleDictionary[2], exampleDictionary[4], 
-      exampleDictionary[8], exampleDictionary[9]];
+    const dateFields = [
+      exampleDictionary[2],
+      exampleDictionary[4],
+      exampleDictionary[8],
+      exampleDictionary[9]
+    ];
     expect(getDateFields(exampleDictionary)).toEqual(dateFields);
   });
 });
 
 describe('getDateFieldsInForm', () => {
   it('returns date fields for the form given', () => {
     const dateFields = [exampleDictionary[8]];
-    expect(getDateFields(exampleDictionary, ["survey"])).toEqual(dateFields);
+    expect(getDateFields(exampleDictionary, ['survey'])).toEqual(dateFields);
   });
 });
 
 describe('isCategoricalField', () => {
   it('returns true for the expected types', () => {
-    const testFields = ['radio', 'dropdown', 'truefalse', 'yesno']
-      .map(type => ({field_type: type}));
+    const testFields = ['radio', 'dropdown', 'truefalse', 'yesno'].map(type => ({
+      field_type: type
+    }));
 
     testFields.forEach(field => {
       expect(isCategoricalField(field)).toBe(true);
     });
 
-    const expectedMapResult = [false, true, false, true, false, true, true, true, false, false ];
+    const expectedMapResult = [
+      false,
+      true,
+      false,
+      true,
+      false,
+      true,
+      true,
+      true,
+      false,
+      false
+    ];
```

Comment:
This also becomes a lot more verbose. I would rather see it break near the column line:

        const expectedMapResult = [false, true, false, true, false, true, true,
          true, false, false];